### ISSUE TITLE
Readme documentation out of date

### ISSUE THREAD

**Issue by legonigel** [mavlink/mavlink#313]
I ran the mavgen script from the command line in Linux (as described in the README.md):

```bash
$python -m pymavlink.generator.mavgen 
```

By default this should generate a python mavlink file, but I get no output.  I ensured that my PYTHONPATH contained the mavlink folder. Running the file gives no error and prints nothing. Maybe the docs just need some updating.

**Comment by vooon** [comment]
Mavgen executable moved, please use:

    python -m pymavlink.tools.mavgen

**Comment by legonigel** [comment]
This should be fixed in the docs. Also, old code should be removed or somehow cleaned up.
This worked for me:
```
$ python -m pymavlink.tools.mavgen
```

**Comment by legonigel** [comment]
See pull request #314.

**Comment by legonigel** [comment]
Fixed in commit 22ab243c8a1b24596ecc2714869333502331b869
